tests/webview/visual/test_home_objects.py

```diff
@@ -0,0 +1,36 @@
+from tests import markers
+from pages.webview.home import Home
+
+APP_NAME = 'CNX HOMEPAGE BOOKS VISUAL VERIFICATION-TEST1'
```

Comment:
We need to come up with a set way to describe the app. I think it's fine if we set this to `Webview` and leave it like that for when this goes into production.